pkg/rangedbapi/api.go

```diff
@@ -165,7 +165,7 @@ func (a *api) saveEvents(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	expectedStreamSequenceNumber := stringToSequenceNumber(r.Header.Get("X-ExpectedStreamSequenceNumber"))
+	expectedStreamSequenceNumber := stringToSequenceNumber(r.Header.Get("ExpectedStreamSequenceNumber"))
```

Comment:
Isn't the usage of `X-` part of the standard for new headers?

Reply:
Not anymore: https://tools.ietf.org/html/rfc6648

I thought about using `RDB-ExpectedStreamSequenceNumber`; however, I highly doubt there will ever be a collision with `ExpectedStreamSequenceNumber`.

Reply:
Wouldn't mind `RDB-*` thought, for a clear and easy way to map headers in proxies, potential tracing in a network and stuff like that.

Why would you do that? it is a matter of time.  🤷🏻